custom-recipes/ocr-text-extraction-dataset/recipe.json

```diff
@@ -48,8 +58,15 @@
             "name": "language",
```

Comment:
I prefer to have distinct parameters depending on the OCR engine so that when you change engine, you get a new empty language parameter that you need to fill again, otherwise, when you change engine, you would still have the other language written and forget to edit it.

I think we can keep the old "language" parameter for tesseract (and "language_easyocr" for easyOCR) and make it extra clear in the Constants by having `LANGUAGE_TESSERACT = "language"` and `LANGUAGE_EASYOCR = "language_easyocr"`